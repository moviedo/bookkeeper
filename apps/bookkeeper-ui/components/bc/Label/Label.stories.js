export default {
  title: "base components / bc-label",
};

const Template = (_args, { argTypes }) => ({
  props: Object.keys(argTypes),
  template: "<bc-label>{{$props.slot}}</bc-label>",
});

export const Default = Template.bind({});
Default.args = {
  slot: "Input Label",
};
