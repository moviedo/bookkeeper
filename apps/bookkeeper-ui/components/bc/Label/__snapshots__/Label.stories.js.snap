// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base components / bc-label Default smoke-test 2`] = `
<label class="text-base font-normal text-slate-600">
  Input Label
</label>
`;
